department-of-veterans-affairs/caseflow
# Connecting scheduled hearings

Resolves #6574 

This PR connects scheduled hearings to the Assign Hearings page. 

To test:
1) go to hearings/schedule/assign
1) Select Central office
1) Confirm hearings are displayed
1) Add a video hearing for an RO
1) Select that RO
1) Confirm hearing is displayed
